tests/functional/pyocf/utils.py

```diff
@@ -176,6 +176,21 @@ def __repr__(self):
     def __eq__(self, other):
         return self.bytes == other.bytes
 
+    def __lt__(self, other):
+        return self.bytes < other.bytes
+
+    def __gt__(self, other):
+        return self.bytes > other.bytes
+
+    def __le__(self, other):
+        return self.bytes <= other.bytes
+
+    def __ge__(self, other):
+        return self.bytes >= other.bytes
+
+    def __ne__(self, other):
+        return self.bytes != other.bytes
+
```

Comment:
Comparisons are already implemented in this class. See lines 85-101 above.

Reply:
:O